pkg/daemons/agent/agent_windows_test.go

```diff
@@ -0,0 +1,26 @@
+// +build windows
+
+package agent
+
+import (
+	"testing"
+
+	"github.com/rancher/k3s/pkg/daemons/config"
+)
+
+func TestCheckRuntimeEndpoint(t *testing.T) {
```

Comment:
Thanks for including a test! @dereknola spent some time getting a base expectation for tests going. Take a look at the etcd package for examples. We're using `gotests` for test generation. We would want to keep consistent.

Reply:
Oh, I didn't see that. I will make that change.